content/docs/1.1.0/snapshots-and-backups/backup-and-restore/set-backup-target.md

```diff
@@ -48,8 +48,7 @@ This page covers the following topics:
 3. Create a Kubernetes secret with a name such as `aws-secret` in the namespace where longhorn is placed(`longhorn-system` by default). For help creating a secret, refer to the [Kubernetes documentation.](https://kubernetes.io/docs/concepts/configuration/secret/) The secret must be created in the `longhorn-system` namespace for Longhorn to access it. Put the following key-value pairs in the secret:
 
     ```shell
-    AWS_ACCESS_KEY_ID: <your_aws_access_key_id>
-    AWS_SECRET_ACCESS_KEY: <your_aws_secret_access_key>
+    kubectl create secret generic aws-secret --from-literal=AWS_ACCESS_KEY_ID=<your_aws_access_key_id> --from-literal=AWS_SECRET_ACCESS_KEY=<your_aws_secret_access_key> -n longhorn-system
```

Comment:
Need to let users know they can choose the secret name (instead of hard coding `aws-secret`).